tests/unit/cython/utils.py

```diff
@@ -0,0 +1,24 @@
+from cassandra.cython_deps import HAVE_CYTHON, HAVE_NUMPY
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest  # noqa
+
+def cyimport(import_path):
+    """
+    Import a Cython module if available, otherwise return None
+    (and skip any relevant tests).
+    """
+    try:
+        return __import__(import_path, fromlist=True)
```

Comment:
This does not work in PyPy. Would ``return __import__(import_path, fromlist=[True])`` have the desired effect? This seems to work in all runtimes I tested.